app/views/comments/create.js.erb

```diff
@@ -0,0 +1 @@
+$("#page_comments").html("<%= j render 'comments', resource: @resource, comment: @comment, comments: @comments, pagy: @pagy %>")
```

Comment:
why need to pass pagy everywhere?

Reply:
Pagination is happening in comments partial so it needs to be passed along

Reply:
On creation of a comment I am re-rendering the comments section